lib/remote/jsonrpcconnection-pki.cpp

```diff
@@ -66,8 +66,14 @@ Value RequestCertificateHandler(const MessageOrigin::Ptr& origin, const Dictiona
 			}
 			logmsg << " signed by our CA.";
 		} catch (const std::exception &ex) {
-			logmsg << " not signed by our CA";
-			if (const unsigned long *openssl_code = boost::get_error_info<errinfo_openssl_error>(ex)) {
+			const unsigned long *openssl_code = boost::get_error_info<errinfo_openssl_error>(ex);
+			if (openssl_code && *openssl_code == X509_V_ERR_CERT_REVOKED) {
+				logmsg << " no longer trusted";
+			} else {
+				logmsg << " not signed by our CA";
+			}
```

Comment:
I'd simply go with some generic wording like "not trusted"/"verification failed"/... here. Anything more specific only duplicates the error message from OpenSSL anyways and is at risk of being inconsistent with error conditions we did not explicitly consider.